src/indentConfig.ml

```diff
@@ -406,11 +406,21 @@ let rec find_conf_file path =
 let local_default ?(path=Sys.getcwd()) () =
   let conf = default in
   let conf, syn, dlink =
-    try
-      let (/) = Filename.concat in
-      let f = (Sys.getenv "HOME") / ".ocp" / "ocp-indent.conf" in
-      if Sys.file_exists f then load ~indent:conf f else conf, [], []
-    with Not_found -> conf, [], []
+    let (/) = Filename.concat in
+    let getenv_f var path =
+      match Sys.getenv var with (* not using getenv_opt because should compile with old versions of OCaml *)
+      | "" -> None
+      | x -> Some ( x / path )
+      | exception Not_found -> None (* not that old *)
+    in
+    match getenv_f "XDG_CONFIG_HOME" ( "ocp" / "ocp-indent.conf" ), (* the XDG way *)
+          getenv_f "HOME" ( ".config" / "ocp" / "ocp-indent.conf" ), (* the XDG alternative *)
```

Comment:
Somehow the XDG spec doesn't say anything about this (or says it in a very indirect manner) but I don't think this is the proper way of doing this. 

The reason is that you can't then disable a set of configuration files by setting your `XDG_CONFIG_HOME` to another directory. With the current code if you do, an exisiting `HOME/.config/ocp/ocp-indent` will be picked up which I think wouldn't be in the spirit of the XDG spec.

I think the right way of doing this is: 

1. Determine a configuration directory `$DIR` according to the XDG spec. If `$DIR/ocp/ocp-indent.conf` exists use this. 
2. If the previous step didn't give anything try (to maintain backward compat) `~/.ocp/ocp-indent.conf`.

Reply:
My bad, fixing this.